builder/vmware/common/ssh.go

```diff
@@ -14,15 +14,10 @@ func CommHost(config *SSHConfig) func(multistep.StateBag) (string, error) {
 		driver := state.Get("driver").(Driver)
 		comm := config.Comm
 
-		// Figure out which protocol we're using to communicate with the
-		// guest, and determine the correct port to handshake with.
-		port := comm.SSHPort
-		if comm.SSHHost != "" {
```

Comment:
It looks like here you've removed the ability for users to override the host using the ssh_host template option. Was that intentional, or a casualty of being hidden in between two chunks of port logic?  We definitely should keep the override, though we can clean it up by writing it out like

```
host := comm.Host()
if host != "" {
 return host, nil
}
```

Reply:
yes. that was a casualty. :-/

Reply:
I just pushed a test to master that should have caught this for you :)

Reply:
heh, ok. added it back. that was mitchellh's code that i mistakenly rm'd.